0200-background.Rmd

```diff
@@ -1,377 +1,375 @@
+
+<br>
+
+### Zymoetz River
+
+Within the Zymoetz Watershed, there are many areas with high fishery values. Steelhead are the most extensively documented fish species in the Zymoetz River watershed. Adults enter the river from JUly to November and then go on to spawn the following year in late spring to early summer. The Zymoetz River is a relatively steep system. Two canyons are located 6.4 and 19.6 kilometers upstream of the Skeena River confluence. These canyons make access to the Zymoetz difficult for pink and chum salmon [@gottesfeld_etal2002ConservingSkeena]. The Zymoetz River is renowned for its aggressive steelhead that have been known to take flies or lures. There is a 50km stretch upstream of Limonite Creek where the Zymoetz River is a designated Class I water. This section is remote and offers high quality fishing opportunities for anglers [@Zymoetz1_AMP.pdf].
+
+<br>
+
+Traditional First Nations use of the upper Zymoetz River watershed by the Gitxsan and Wet'suwet'en people differed between community sites, residences, and fish houses, and was large and diverse. From the upper to lower Zymoetz River and to the Skeena River, a significant ancient grease trail connected, with a branch track forking through Limonite Creek and flowing down the Telkwa River. The fishery used a weir at the mouth of McDonell Lake and spears at Six Mile Flats, near Dennis Lake. There is no information on native fisheries on the lower Zymoetz River. The Zymoetz is considered to be one of the top ten steelhead rivers in BC [@gottesfeld_etal2002ConservingSkeena].
+
+<br>
+
+### Kispiox River
+
+Kispiox River salmon are a important food source and cultural symbol for the Gitxsan people with sockeye and coho historically the two most significant species. Gitangwalk and Lax Didax, two significant villages that were both abandoned in the early 1900s, were situated on the Kispiox in such a way as to block the sockeye and coho salmon's upstream migration to the Upper Kispiox River spawning grounds providing opportunities to gather and preserve a significant amount of high-quality food over relatively short time periods [@gottesfeld_etal2002ConservingSkeena]. About 100 km of the mainstem and 300 km of the tributary streams in the Kispiox River Watershed are considering high value fish habitat as they are a migration route and support spawning and rearing for many fish species. The Kispiox fisheries supports both recreational and commercial fishing while also enhancing the ecology, nutrient regime, and structural diversity of the drainage. Since 1992, sockeye and coho escapements from the Kispiox Watershed have been documented by the Gitxsan Watershed Authorities as they creates strong cultural, economic, and symbolic ties for the local communities [@gottesfeld_etal2002ConservingSkeena].
```

Comment:
Regarding your comment on the PR, I'm not sure what general area it occurs. Were you referring to the escapements? Couldn't find any more info on that.

Reply:
If we are saying "About 100 km of the mainstem and 300 km of the tributary streams in the Kispiox River Watershed are considering high value fish habitat" then the logical question is "which 100km of mainstem and which 300km of tributaries"?  If we don't know then we shouldn't write it down. I can revise.

Reply:
same idea for Class 1...

Reply:
Just checked the source I got that from and realised I changed the wording on that sentence and unintentionally changed the meaning of it. Here's the original "The Kispiox River Watershed is composed of approximately 100 km of mainstem and 300 km of tributary streams that are considered high value fish habitat." So it's referring to the whole watershed.